src/plumpy/processes.py

```diff
@@ -990,16 +989,16 @@ def broadcast_receive(
             self.pid,
             subject,
             _comm,
-            body,
+            msg,
         )
 
         # If we get a message we recognise then action it, otherwise ignore
         if subject == process_comms.Intent.PLAY:
             return self._schedule_rpc(self.play)
         if subject == process_comms.Intent.PAUSE:
-            return self._schedule_rpc(self.pause, msg=body)
+            return self._schedule_rpc(self.pause, msg_text=msg.get(process_comms.MESSAGE_KEY, None))
```

Comment:
I found this change a  bit hard to read & understand.
What does it do, and this has to change?

Reply:
This is actually similar to the `_perform_action` of aiida-core, here the `msg_text` is passing as the parameter of `self.pause`. Together they form a callback. 
I can do `functools.partial(self.pause, msg_text='')`, do you think that is better? 

Here for `broadcast_receive` and `message_receieve` the massege passed in should be a message as a whole, it makes less sense to accept `msg_text` and other argument passing to command specifically.